models/index.js

```diff
@@ -0,0 +1,12 @@
+require('dotenv').config();
+const mongoose = require('mongoose');
+const schema = require('./schema/userSchema');
```

Comment:
The `schema` variable name is very general and will be confusing when we add more schemas into the DB.